src/requirements/data/templates/ece.ts

```diff
@@ -0,0 +1,86 @@
+import { Template } from '@/requirements/types';
+
+const eceTemplate: Template = [
+  {
+    reqGroup: 'UNI',
+    name: 'Physical Education',
+    placeholderSemesters: [1, 2],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Mathematics',
+    placeholderSemesters: [1, 2, 3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Physics',
+    placeholderSemesters: [2, 3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Chemistry',
+    placeholderSemesters: [1],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'First-Year Writing Seminars',
+    placeholderSemesters: [1, 2],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Advanced Programming/Computer Engineering',
+    placeholderSemesters: [3],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'ENGRD 2300',
+    placeholderSemesters: [3],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Core Courses',
```

Comment:
This template does not match the reqs file, but I think you have the most up to date information. @einc it looks like the current ECE requirement just has core courses ECE 2300 and ECE 2720, should we update the reqs file?

![image](https://user-images.githubusercontent.com/25535093/142491341-cd280bfe-c5a1-4147-95b6-239d2585b47c.png)

Reply:
Yea, would you guys mind updating the ECE file as well? They haven't been updated in a while, thank you!

Reply:
Looks like the ECE core courses changed in 2021 - similarly to electives, will probably want toggleable requirement depending on class year!